src/proto/mgmt/svc.proto

```diff
@@ -43,7 +43,6 @@ message JoinReq {
 	uint64 incarnation = 9; 	// rank incarnation
 	repeated string secondary_uris = 10; // URIs for any secondary providers
 	repeated uint32 secondary_nctxs = 11; // CaRT context count for each secondary provider
-	bool check_mode = 12; 		// rank started in check mode
```

Comment:
Oops, this stuff shouldn't have been removed. This is important for the CR functionality. I will re-add it and regenerate the files.

Reply:
Looks like the protobuf files had not been generated with these changes yet. I saw no changes when I regenerated them after re-adding the check fields.